airflow/jobs/scheduler_job_runner.py

```diff
@@ -1822,9 +1822,30 @@ def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
                                 ),
                             )
                         )
```

Comment:
Can you update the wording in the logging and docstring above, given the new behaviour.

Reply:
Yes I've created separate logs for rescheduling and failing of tasks